depthai_nodes/ml/parsers/vehicle_attributes.py

```diff
@@ -0,0 +1,61 @@
+from typing import List
+
+import depthai as dai
+
+from ..messages.creators import create_multi_classification_message
+
+
+class MultiClassificationParser(dai.node.ThreadedHostNode):
```

Comment:
Can we remove any mention of vehicle model in this file - file name, docstrings, variable names - as now this parser should be general for all multi-classification models right?